src/main/java/seedu/clinic/logic/commands/AddCommand.java

```diff
@@ -3,65 +3,97 @@
 import static java.util.Objects.requireNonNull;
 import static seedu.clinic.logic.parser.CliSyntax.PREFIX_ADDRESS;
 import static seedu.clinic.logic.parser.CliSyntax.PREFIX_EMAIL;
-import static seedu.clinic.logic.parser.CliSyntax.PREFIX_NAME;
 import static seedu.clinic.logic.parser.CliSyntax.PREFIX_PHONE;
-import static seedu.clinic.logic.parser.CliSyntax.PREFIX_TAG;
+import static seedu.clinic.logic.parser.CliSyntax.PREFIX_REMARK;
+import static seedu.clinic.logic.parser.CliSyntax.PREFIX_SUPPLIER_NAME;
+import static seedu.clinic.logic.parser.CliSyntax.PREFIX_WAREHOUSE_NAME;
 
 import seedu.clinic.logic.commands.exceptions.CommandException;
 import seedu.clinic.model.Model;
 import seedu.clinic.model.supplier.Supplier;
+import seedu.clinic.model.warehouse.Warehouse;
 
 /**
- * Adds a supplier to the CLI-nic app.
+ * Adds a supplier/warehouse to the CLI-nic app.
  */
 public class AddCommand extends Command {
 
     public static final String COMMAND_WORD = "add";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a supplier to the CLI-nic app. "
-            + "Parameters: "
-            + PREFIX_NAME + "NAME "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a supplier/warehouse to CLI-nic.\n"
+            + "Supplier Parameters: "
+            + PREFIX_SUPPLIER_NAME + "NAME "
             + PREFIX_PHONE + "PHONE "
             + PREFIX_EMAIL + "EMAIL "
-            + PREFIX_ADDRESS + "ADDRESS "
-            + "[" + PREFIX_TAG + "TAG]...\n"
+            + PREFIX_REMARK + "REMARK "
```

Comment:
I think there should be a \n here

Reply:
Hmmm... Okay I'm including these line breaks,